src/create-context.js

```diff
@@ -23,14 +22,21 @@ export function createContext(defaultValue) {
 	let ctx = { [id]: null };
 
 	function initProvider(comp) {
-		let subs = [];
+		const subs = [];
 		comp.getChildContext = () => {
 			ctx[id] = comp;
 			return ctx;
 		};
-		comp.componentDidUpdate = () => {
-			let v = comp.props.value;
-			subs.map(c => v!==c.context && (c.context = v, enqueueRender(c)));
+		comp.shouldComponentUpdate = props => {
+			if (props.value!==comp.value) {
+				subs.map(c => {
+					// Check if still mounted
+					if (c._parentDom) {
+						c.context = props.value;
```

Comment:
Removing both the `if`-statement and the `c.context` assignment has no effect on the test outcomes. Maybe we're missing something?

Reply:
Parent -> context -> noUpdate-> consumer could have skewed results without it, no?

Reply:
I tested this and well it does not fail.... No tests fails + my additional test for that scenario also succeeds on top that saves an additional 7B